source/crud/tests/v2/find-allowdiskuse.yml

```diff
@@ -0,0 +1,50 @@
+runOn:
+  - { minServerVersion: "4.4.0" }
+
+collection_name: &collection_name 'test_find_allowdiskuse'
+
+tests:
+  -
+    description: "Find does not send allowDiskuse when unspecified"
+    operations:
+      -
+        object: collection
+        name: find
+        arguments:
+          filter: { }
+    expectations:
+      -
+        command_started_event:
+          command:
+            find: *collection_name
+            allowDiskUse:
+  -
+    description: "Find sends allowDiskuse false"
```

Comment:
Since we're not testing the "true" case, consider renaming to "Find sends allowDiskUse when specified"

Reply:
This is testing the false case. The true case IS tested below.

I've reworded slightly to be more explicit.